orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/pipeline/PipelineStage.java

```diff
@@ -71,6 +72,17 @@ public void taskGraph(@Nonnull StageExecution stage, @Nonnull TaskNode.Builder b
     }
   }
 
+  @Override
+  public void afterStages(@Nonnull StageExecution parent, @Nonnull StageGraphBuilder graph) {
```

Comment:
That parameter parent is really the current stage itself and not the parent. So, no calling it parent will reduce some confusion while reading the code.

Reply:
done